### PR TITLE
Add TV Time

### LINKED ISSUE #2952: Add TV Time icon
**Name:** TV Time
**Website:** https://www.tvtime.com/
**Official resources for icon and color:** I got a SVG file (enclosed) directly from TV Time support team.
  

[tvt-logo.zip](https://github.com/simple-icons/simple-icons/files/4497051/tvt-logo.zip)

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
TV Time

**Issue:** Closes #2952
**Alexa rank:** [~23.2k](https://www.alexa.com/siteinfo/tvtime.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
~I'm not _entirely_ sure what to use for the source here, as the only source we currently have for an official vector image is one of our own issues. Should we go with [their website](https://www.tvtime.com/), despite the absence of a vector logo?~

Possible alternative treatment & colour (or maybe one of the other greys from their website):

### CODE REVIEW

**Comment by adamrusted** [comment]
Think their website is sufficient as a source, as anyone updating this icon _should_ look back through previous PRs / Issues to see sources that were used, and reasoning behind them. To me, the yellow seems more appropriate as a Hex, as it's their 'CTA' colour on the website and - by the looks of it - the app. I also think the boxed `T` is better, as it more resembles the 'App Icon' style logo they use throughout.

**Comment by PeterShaggyNoble** [comment]
Another point about the source is that the GH is stale; looking at it won't tell you whether or not they have a new logo in the future. So ... source updated.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PeterShaggyNoble!

Will await some feedback by other @simple-icons/maintainers before merging, however.

**Review by adamrusted — APPROVED** [review]
Merging this, as it's over 10 days with no additional feedback. Thanks @PeterShaggyNoble 💯

**Pull request merged**
